pkgs/development/tools/build-managers/dub/default.nix

```diff
@@ -1,14 +1,12 @@
-{ stdenv, fetchFromGitHub, curl, dmd, gcc }:
+{ stdenv, fetchurl, curl, dmd, gcc }:
 
 let version = "0.9.24"; in
 stdenv.mkDerivation {
   name = "dub-${version}";
 
-  src = fetchFromGitHub {
-    sha256 = "52ea88d075f3a43c84d8cdc3a8683bcc8133b7db";
-    rev = "v${version}";
-    repo = "dub";
-    owner = "dlang";
+  src = fetchurl {
```

Comment:
`fetchFromGitHub` is a thin wrapper around `fetchzip`, which uses the hash of the archive's contents instead of the archive itself; this is why it is more stable than `fetchurl`. When github arbitrarily decides to change the codec or compression levels for their generated tarballs, `fetchFromGitHub` will continue to work, whereas `fetchurl` will fail.